pkg/api/latest/latest.go

```diff
@@ -30,8 +31,21 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
 )
 
+func init() {
+	version := os.Getenv("KUBE_API_LATEST_VERSION")
```

Comment:
Done. Updated the env variable to specify the complete list.
As discussed keeping it as an env flag instead of a command line flag, since command line flags are processed after init methods are called.